Source/Android/jni/AndroidCommon/IDCache.cpp

```diff
@@ -474,6 +487,12 @@ JNIEXPORT jint JNI_OnLoad(JavaVM* vm, void* reserved)
   s_riivolution_patches_pointer = env->GetFieldID(riivolution_patches_class, "mPointer", "J");
   env->DeleteLocalRef(riivolution_patches_class);
 
+  const jclass wii_update_cb_class =
+      env->FindClass("org/dolphinemu/dolphinemu/utils/WiiUpdateCallback");
+  s_wii_update_cb_class= reinterpret_cast<jclass>(env->NewGlobalRef(wii_update_cb_class));
```

Comment:
``s_wii_update_cb_class =``

Reply:
lint will take care of it, so you can save a little bit of time by skipping over these kinds of things when reviewing a PR.